bibtex-actions.el

```diff
@@ -536,15 +520,18 @@ With prefix, rebuild the cache before offering candidates."
 
 (defun bibtex-actions-run-default-action (keys)
   "Run the default action `bibtex-actions-default-action' on KEYS."
-  (funcall bibtex-actions-default-action
-           (if (stringp keys) (split-string keys " & ") keys)))
+  (let* ((keys-raw (if (consp keys) (cdr keys) keys))
+         (keys-list
+          ;; Make sure we split the crm-string we provide 'embark'.
+          (if (stringp keys-raw) (split-string keys-raw) " & " keys-raw)))
+    (funcall bibtex-actions-default-action keys-list)))
 
 ;;;###autoload
 (defun bibtex-actions-dwim ()
   "Run the default action on citation keys found at point."
   (interactive)
   (if-let ((keys (bibtex-actions-citation-key-at-point)))
```

Comment:
```suggestion
  (if-let ((keys (cdr (bibtex-actions-citation-key-at-point))))
```

`bibtex-actions-citation-key-at-point` return a cons as the embark target, but `bibtex-actions-dwim` only needs the cdr part, i.e. citation keys.

Reply:
This alone doesn't fix it for me.